catroid/src/main/java/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.kt

```diff
@@ -0,0 +1,767 @@
+        soundBrickList.add(PlayNoteForBeatsBrick(BrickValues.DEFAULT_NOTE, BrickValues.PAUSED_BEATS_INT))
+        soundBrickList.add(PlayDrumForBeatsBrick(BrickValues.PAUSED_BEATS_INT))
+        soundBrickList.add(SetTempoBrick(BrickValues.DEFAULT_TEMPO))
+        soundBrickList.add(ChangeTempoByNBrick(BrickValues.CHANGE_TEMPO))
+        soundBrickList.add(PauseForBeatsBrick(BrickValues.PAUSED_BEATS_FLOAT))
+        return soundBrickList
+    }
+
+    protected open fun setupLooksCategoryList(context: Context, isBackgroundSprite: Boolean): List<Brick> {
+        val looksBrickList: MutableList<Brick> = ArrayList()
+        if (!isBackgroundSprite) {
+            looksBrickList.add(SetLookBrick())
+            looksBrickList.add(SetLookByIndexBrick(BrickValues.SET_LOOK_BY_INDEX))
+        }
+        looksBrickList.add(NextLookBrick())
+        looksBrickList.add(PreviousLookBrick())
+        looksBrickList.add(SetSizeToBrick(BrickValues.SET_SIZE_TO))
+        looksBrickList.add(ChangeSizeByNBrick(BrickValues.CHANGE_SIZE_BY))
+        looksBrickList.add(HideBrick())
+        looksBrickList.add(ShowBrick())
+        looksBrickList.add(AskBrick(context.getString(R.string.brick_ask_default_question)))
+        if (!isBackgroundSprite) {
+            looksBrickList.add(SayBubbleBrick(context.getString(R.string.brick_say_bubble_default_value)))
+            looksBrickList.add(SayForBubbleBrick(context.getString(R.string.brick_say_bubble_default_value), 1.0f))
+            looksBrickList.add(ThinkBubbleBrick(context.getString(R.string.brick_think_bubble_default_value)))
+            looksBrickList.add(ThinkForBubbleBrick(context.getString(R.string.brick_think_bubble_default_value), 1.0f))
+        }
+        looksBrickList.add(ShowTextBrick(BrickValues.X_POSITION, BrickValues.Y_POSITION))
+        looksBrickList.add(ShowTextColorSizeAlignmentBrick(BrickValues.X_POSITION, BrickValues.Y_POSITION, BrickValues.RELATIVE_SIZE_IN_PERCENT, BrickValues.SHOW_VARIABLE_COLOR))
+        looksBrickList.add(SetTransparencyBrick(BrickValues.SET_TRANSPARENCY))
+        looksBrickList.add(ChangeTransparencyByNBrick(BrickValues.CHANGE_TRANSPARENCY_EFFECT))
+        looksBrickList.add(SetBrightnessBrick(BrickValues.SET_BRIGHTNESS_TO))
+        looksBrickList.add(ChangeBrightnessByNBrick(BrickValues.CHANGE_BRIGHTNESS_BY))
+        looksBrickList.add(SetColorBrick(BrickValues.SET_COLOR_TO))
+        looksBrickList.add(ChangeColorByNBrick(BrickValues.CHANGE_COLOR_BY))
+        looksBrickList.add(ClearGraphicEffectBrick())
+        looksBrickList.add(WhenBackgroundChangesBrick())
+        looksBrickList.add(SetBackgroundBrick())
+        looksBrickList.add(SetBackgroundByIndexBrick(BrickValues.SET_LOOK_BY_INDEX))
+        looksBrickList.add(SetBackgroundAndWaitBrick())
+        looksBrickList.add(SetBackgroundByIndexAndWaitBrick(BrickValues.SET_LOOK_BY_INDEX))
+        if (!ProjectManager.getInstance().currentProject.isCastProject) {
```

Comment:
please use injected ProjectManager
```suggestion
        if (!projectManager.currentProject.isCastProject) {
```